e2e/nomostest/new.go

```diff
@@ -471,7 +471,12 @@ func setupTestCase(nt *NT, opts *ntopts.New) {
 	// understand the Repo and RootSync types as ConfigSync is now installed.
 	nt.RenewClient()
 
-	// Create the RootSync if it doesn't exist, and wait for it to be Synced.
+	// Initialize the base RootSync using SSA for field management
+	rs := RootSyncObjectV1Beta1FromRootRepo(nt, configsync.RootSyncName)
+	if err := nt.KubeClient.Apply(rs); err != nil {
+		nt.T.Fatal(err)
+	}
+	// Wait for Config Sync to be ready and the base RootSync to be synced.
 	if err := WaitForConfigSyncReady(nt); err != nil {
```

Comment:
`WaitForConfigSyncReady` calls `ValidateMultiRepoDeployments`, which creates the RootSync again. Should we remove the CREATE call from `ValidateMultiRepoDeployments`?

Reply:
Yes I think we can probably remove the CREATE call, however `WaitForConfigSyncReady` is called by other code paths - so I figured it's safer to just start with this change since we probably want to cherry pick